crates/ruff_workspace/src/resolver.rs

```diff
@@ -303,6 +303,7 @@ pub fn resolve_configuration(
     pyproject: &Path,
     relativity: Relativity,
     transformer: &dyn ConfigurationTransformer,
+    provenance: Option<&ConfigurationProvenance>,
```

Comment:
Can we update the function documentation to indicate what does `None` mean here?